deploy-edpm.yml

```diff
@@ -6,6 +6,14 @@
 # openstack-k8s-operators/install_yaml repository in order to leverage its
 # own methods.
 
+- name: Remove status flag
```

Comment:
@cjeanner I'm not sure if that's the right place for removing and injecting the flag. Is there any better (more generic) location?

Reply:
Sounds like the right place. Another way would be with a hook, but it would require people to know about it and inject it. Here, it's written in plain in the main entrypoint playbook, so it should be all fine.

We can't really use the sub-playbooks (well, we can clean in 01-bootstrap), since some aren't always included, such as the 08-run-tests.yml.. So I don't have better alternative.